### ISSUE TITLE
Generic `__getattr__` fallback ignored with multiple inheritance

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#2897]
**Describe the bug**
When inheriting from a generic class which defines `__getattr__` and which is not the nearest parent of the new class, the `__getattr__` method is not inherited and attempting to access arbitrary attributes produces a type error.

**To Reproduce**
Type check the snippet below.

**Expected behavior**
No error would be produced.

**Screenshots or Code**

```python
# pyright: strict

from typing import Generic, TypeVar

T = TypeVar("T")

class Foo:
    pass

class Bar(Generic[T]):
    def __getattr__(self, key: str) -> T: ...

class Baz(Foo, Bar[T]):
    pass

a = Bar[object]().a  # Ok
Baz[object]().a  # Member "a" is unknown

```

**VS Code extension or command-line**
Either, 1.1.212

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.213, which I just published. It will also be included in the next release of pylance.
